_data/simple-icons.json

```
@@ -8960,6 +8960,11 @@
             "hex": "FFE800",
             "source": "https://www.sahibinden.com/favicon.ico"
         },
+        {
+            "title": "Sailfish OS",
+            "hex": "053766",
+            "source": "https://sailfishos.org/content/uploads/2021/02/SF4.svg"
```

```suggestion
            "source": "https://sailfishos.org"
```

Can you change this, please.